src/Helmet.js

```diff
@@ -134,6 +142,22 @@ const updateTitle = title => {
     document.title = title || document.title;
 };
 
+const updateHtmlAttributes = (attributes) => {
+    const htmlTag = document.getElementsByTagName("html")[0];
+
+    const oldAttributeCount = htmlTag.attributes.length;
+    if (oldAttributeCount) {
+        for (let i = oldAttributeCount - 1; i >= 0; i--) {
+            htmlTag.removeAttribute(htmlTag.attributes[i].name);
```

Comment:
Ah, ok, with the other tags we are searching for tags that have the `data-react-helmet` attribute.  Hmm, don't know a good way to do that in this case, so this is fine we remove them then.  This will force everyone to manage all their html tag attributes in Helmet, which I don't see a downside with at the moment. :-)